packages/web-components/fast-foundation/src/flipper/flipper.template.ts

```diff
@@ -1,6 +1,27 @@
-import { ElementViewTemplate, html, when } from "@microsoft/fast-element";
+import { ElementViewTemplate, html } from "@microsoft/fast-element";
 import type { FASTFlipper, FlipperOptions } from "./flipper.js";
-import { FlipperDirection } from "./flipper.options.js";
+import type { FlipperDirection } from "./flipper.options.js";
+
+const templateCache = {};
+
+function setFlipperTemplateByDirection(
+    direction: FlipperDirection,
+    options: FlipperOptions
+) {
+    let existing = templateCache[direction];
+
```

Comment:
I think this means that flipperTemplate can't be re-used with different options, since the cache is only based on direction and not direction *and* options. That doesn't seem ideal, though I acknowledge re-using this template is probably an edge case.

Reply:
Even in the case that other options change, this is just for the internal portion which depends on direction and not the whole thing - though I may be missing the nuance here.

Reply:
Oh! That is a good point @nicholasrice. @chrisdholt We can fix this by moving the cache and helper function into the function body that creates the template. That will then give you a cache per template, enabling the function to be called multiple times with different options to create variations.

Reply:
So, basically, move lines 5 - 24 into the function right before line 33.